src/foobar/rest/serializers/purchase.py

```diff
@@ -33,5 +35,40 @@ def as_purchase_kwargs(self):
         }
 
 
+class PurchaseItemSerializer(serializers.Serializer):
+    qty = serializers.IntegerField()
+    product_id = serializers.CharField()
+
+
 class PurchaseSerializer(serializers.Serializer):
+    """Serializes a purchase, as retrieved from the foobar API call:
+        `get_purchase()`
+    """
+    id = serializers.UUIDField(allow_null=False)
     amount = MoneyField()
+    status = IntEnumField()
+
+    def to_representation(self, instance):
+        purchase_obj = super().to_representation(instance[0])
```

Comment:
These `instance[x]` took me a while to process. Wouldn't it be better to deconstruct the instance-tuple instead, just to make it more explicit?
```
purchase, items = instance
```